crates/iceberg/Cargo.toml

```diff
@@ -67,6 +67,8 @@ flate2 = { workspace = true }
 fnv = { workspace = true }
 futures = { workspace = true }
 itertools = { workspace = true }
+log = { workspace = true }
+miniz_oxide = "0.8"
```

Comment:
We manage dependencies version in root workspace.

Reply:
done.